site/zenodo_rdm/legacy/serializers/__init__.py

```diff
@@ -33,6 +33,12 @@ def unwrap_hits(self, data, many, **kwargs):
         return data.get("hits", {}).get("hits", [])
 
 
+class ZenodoListSchema(BaseListSchema):
+    """Zenodo top-level List schema."""
+
+    sortBy = fields.Field(load_only=True)
```

Comment:
question: do you know where the `sort` parameter is loaded/dumped now?

Reply:
question: why do we need to reimplement this here? it's already being done in `flask-resources`

Reply:
> question: do you know where the `sort` parameter is loaded/dumped now?

@slint  by sort, do you mean the `sortBy` ? I didn't track it, thinking it might be being used somewhere as it is allowed everywhere so I set load_only=True, should I just override and return missing?

Reply:
> question: why do we need to reimplement this here? it's already being done in `flask-resources`

@anikachurilova  this is in response to this comment https://github.com/zenodo/zenodo-rdm/pull/665#discussion_r1424300682

Reply:
> > question: do you know where the `sort` parameter is loaded/dumped now?
> 
> @slint by sort, do you mean the `sortBy` ? I didn't track it, thinking it might be being used somewhere as it is allowed everywhere so I set load_only=True, should I just override and return missing?

I mean if one passes now `GET /api/records?sort=mostrecent`, I would expect:

1. the parameter `sort=mostrecent` is passed down to the service to return records sorted by most recent first
2. in the response I see in the `links`, `self: /api/records?sort=mostrecent&page=1&size=25, ...`

I don't see anywhere in the schema a field interacting with the `?sort=...` parameter